docs/dgfit/dustgrain_props.rst

```diff
@@ -0,0 +1,108 @@
+#####################
+Dust grain properties
+#####################
+
+There are several plots that show the effective grain properties of the used dustgrains.
+In all plots, it is possible to transform the particles and data to the observed data that was used.
+
+To plot the extinction coefficients, albedo, scattering phase function and emission of the used dustgrains (default is astro-silicates) in function of the grain size, use command
+
+    dgplot_dustgrains
```

Comment:
It would be good if these were identified with formatting as something that goes on the command line.

This can be done with the following above each case like this.  Note there needs to be a blank line between this command and the shell command. 

`.. code-block:: console`